[2024-05-09T12:49:16.047+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-09T12:49:16.073+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dataset_consumes_1.consuming_1 manual__2024-05-09T12:49:12.485481+00:00 [queued]>
[2024-05-09T12:49:16.082+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dataset_consumes_1.consuming_1 manual__2024-05-09T12:49:12.485481+00:00 [queued]>
[2024-05-09T12:49:16.084+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-09T12:49:16.099+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): consuming_1> on 2024-05-09 12:49:12.485481+00:00
[2024-05-09T12:49:16.107+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=136) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-09T12:49:16.109+0000] {standard_task_runner.py:63} INFO - Started process 138 to run task
[2024-05-09T12:49:16.108+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dataset_consumes_1', 'consuming_1', 'manual__2024-05-09T12:49:12.485481+00:00', '--job-id', '3', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_datasets.py', '--cfg-path', '/tmp/tmplf93cdd7']
[2024-05-09T12:49:16.110+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask consuming_1
[2024-05-09T12:49:16.158+0000] {task_command.py:426} INFO - Running <TaskInstance: dataset_consumes_1.consuming_1 manual__2024-05-09T12:49:12.485481+00:00 [running]> on host 605c16266d54
[2024-05-09T12:49:16.254+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dataset_consumes_1' AIRFLOW_CTX_TASK_ID='consuming_1' AIRFLOW_CTX_EXECUTION_DATE='2024-05-09T12:49:12.485481+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-09T12:49:12.485481+00:00'
[2024-05-09T12:49:16.260+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-09T12:49:16.261+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-09T12:49:16.262+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sleep 5']
[2024-05-09T12:49:16.280+0000] {subprocess.py:86} INFO - Output:
[2024-05-09T12:49:21.292+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-09T12:49:21.294+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-09T12:49:21.319+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py:1564: RemovedInAirflow3Warning: Passing 'execution_date' to 'TaskInstance.xcom_push()' is deprecated.
  context["ti"].xcom_push(key=key, value=value, execution_date=execution_date)

[2024-05-09T12:49:21.343+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dataset_consumes_1, task_id=consuming_1, run_id=manual__2024-05-09T12:49:12.485481+00:00, execution_date=20240509T124912, start_date=20240509T124916, end_date=20240509T124921
[2024-05-09T12:49:21.398+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-09T12:49:21.415+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-09T12:49:21.420+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
